draft-pauly-adaptive-dns-privacy.md

```diff
@@ -261,6 +260,12 @@ expected to support acting as a target for Oblivious DoH. A client MUST issue at
 least one query that is targeted at the server through a proxy before sending direct queries
 to the server.
 
+Designated DoH Servers are expected to act both as Oblivious Proxies and as Oblivious Targets
+to ensure that clients have sufficient options for preserving privacy using Oblivious DoH.
+Oblivious Targets are expected to act as Oblivious Proxies to ensure that no Oblivious DoH server
+can act as only a target (thus being able to see patterns in name resolution, which might have
+value to a resolver) and require other servers to take on a disproportionate load of proxying.
```

Comment:
We should probably also recommend some client behavior if certain targets are not willing to proxy. Perhaps, "Clients SHOULD stop using Oblivious Targets in the absence of proxy support"?

Reply:
I think this is already covered by the beginning of the section, where the clients MUST whitelist the servers by validating that they can both be the proxy and target. You should never use a server as a target if it isn't a proxy too.